packages/databyss-ui/editor/EditorBlock.js

```diff
@@ -86,6 +102,7 @@ export const EditorBlock = ({ children, node }) => {
 
   const _children = (
     <View
+      ref={blockEl}
```

Comment:
This can be cleaned up by using the callback form of `ref`. This avoids the need for lines 58-70.

```jsx
<View 
  ref={ref => pageContext && pageContext.registerBlockRef(node.key, ref)}
>
```